distribution/docs/src/main/resources/content/_architectures/_eventing/eventing-intro.adoc

```diff
@@ -1,6 +1,9 @@
-:title: Eventing Intro
-:type: eventing
+:title: Eventing
+:type: architectureIntro
 :status: published
+:children:
+:order: 07
+:summary: Introduction to the Migration API.
```

Comment:
This file should no longer say `After reading this section, you will be able to:` because it no longer comes right before the Subscriptions section.

Reply:
fixed.